packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -18,6 +18,7 @@ import {
 } from './BaseWrapper'
 
 export interface Validator {
+  name: string
```

Comment:
Not sure if this is a "Validator" thing... instead what about "fetching the account" on the cli when you want to show the Validator details?

I don't know if this sounds like something any contractKit user would want, maybe just the cli.

Reply:
hehehe.. I see that me and @asaj are thinking completely opposite things!.. 
It's not a big issue so either way work